src/app/static/src/js/components/runWorkflow/runWorkflowSummary.vue

```diff
@@ -1,48 +1,186 @@
 <template>
     <div>
         <h2 id="summary-header">Summary</h2>
-        <span>Your workflow contains {{ reportCount }}:</span>
-        <ul class="styled">
-            <li v-for="report in workflowMetadata.reports">
-                {{ report.name }}
-            </li>
-        </ul>
+        <div id="summary-warning" class="row mt-3" v-if="hasMissingDependencies">
+            <div class="col-auto">
+                <alert-triangle-icon size="2x" stroke="red" class="custom-class"/>
+            </div>
+            <div class="col-auto">
+                <span class="d-inline-block pb-2"> Some reports depend on the latest version of other reports that are not included in your workflow:</span>
+                <div v-for="(missing_dependencies, report) in dependencies.missing_dependencies">
+                    <span v-if="missing_dependencies.length" class="font-weight-bold"> {{ report }}</span>
+                    <ul v-for="missing_dependency in missing_dependencies" class="styled">
+                        <li>{{ missing_dependency }}</li>
+                    </ul>
+                </div>
+            </div>
+            <hr>
+        </div>
+        <div id="summary-timeline" class="mt-3">
+            <div class="row">
+                <div class="col-12">
+                    <div class="summary">
+                        <div v-for="report in workflowMetadata.reports" class="single-timeline-area">
+                            <div class="timeline-date wow fadeInLeft" data-wow-delay="0.1s"
+                                 style="visibility: visible; animation-delay: 0.1s; animation-name: fadeInLeft;">
+                            </div>
+                            <div class="d-inline-block" v-tooltip="reportInfo(report)">
+                                <h5>{{ report.name }}
+                                    <span>
+                                        <info-icon size="1.2x" stroke="grey" class="custom-class"/>
+                                    </span>
+                                </h5>
+                            </div>
+                            <div class="row">
+                                <div class="col-12 col-md-6 col-lg-4">
+                                    <div class="single-timeline-content parameters-bg-color d-flex wow fadeInLeft"
+                                         data-wow-delay="0.3s"
+                                         style="visibility: visible; animation-delay: 0.3s; animation-name: fadeInLeft;">
+                                        <div class="timeline-text">
+                                            <span class="text-muted d-inline-block">Parameters</span>
+                                            <div v-if="!!paramSize(report)">
+                                                <p v-for="(value, key) in report.params">{{ key }}: {{ value }}</p>
+                                                <div v-if="paramSize(report) > 3">
```

Comment:
So it looks like this is going to show an expandable view labelled 'Show default..' if there are more than three parameters, but hiding the defaults isn't about the number of total parameters in the report, but about always hiding parameters with default values, and showing paramters with non-default values. We should be able to get the default parameter values to compare to from the runnable report list (this will be easier once we have Vuex but the summary view might need to fetch that for now...). Implementing this feels like it should be a ticket in itself!

Reply:
This feels like a ticket, I will create one for this as well.